aiohttp/server.py

```diff
@@ -103,6 +106,14 @@ def __init__(self, *, loop=None,
         self._timeout = timeout  # slow request timeout
         self._loop = loop if loop is not None else asyncio.get_event_loop()
 
+        parser_kwargs = {}
```

Comment:
These lines look like python2-ish.
Is there any reason to don't use keyword-only constructor parameters with reasonable default values?